server/borsen/io/feed_parsers/ritzau.py

```diff
@@ -25,8 +26,23 @@ def __init__(self):
         self.default_mapping.update({"anpa_category": {"xpath": "section/text()", "filter": self._category_filter}})
 
     def _category_filter(self, category):
-        categories = [cat.strip() for cat in category.split(",") if cat.strip()]
-        return [{"name": cat, "qcode": cat.lower()} for cat in categories]
+        voc_categories = superdesk.get_resource_service("vocabularies").find_one(req=None, _id="categories")
+
+        if voc_categories:
+            categories_cv = {
+                i["ritzau_section_id"].lower(): i for i in voc_categories["items"] if "ritzau_section_id" in i
+            }
+        else:
+            categories_cv = {}
+
+        categories = [cat.strip().lower() for cat in category.split(",") if cat.strip()]
+
+        populated_categories = []
+        for cat in categories:
+            match = categories_cv.get(cat)
+            if match:
+                populated_categories.append({"name": match["name"], "qcode": match["qcode"]})
```

Comment:
would be good to alsa add translations if there are some, I think there is some helper method in the service to get CV item for use in the article